packages/neuron-ui/src/widgets/ScreenButtonRow/index.tsx

```diff
@@ -1,5 +1,27 @@
 import styled from 'styled-components'
 
+const RightScreenButtonRow = styled.div`
+  display: flex;
+  padding-top: 30px;
+  justify-content: flex-end;
+
+  button {
+    margin-left: 20px;
+  }
+`
+
+const LeftScreenButtonRow = styled.div`
+  display: flex;
+  padding-top: 30px;
+  justify-content: flex-start;
+
+  button {
+    margin-right: 20px;
+  }
+`
+
+export { RightScreenButtonRow, LeftScreenButtonRow }
```

Comment:
These two kinds of layout could be implemented by extending `ScreenButtonRow` with an `align` attribute.

Reply:
Indeed. Will change to that.